src/assets/img/Layer2.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="26.325" height="27" viewBox="0 0 26.325 27">
```

Comment:
**Question**: Are we using this image anywhere?

Reply:
It is used as the background of the header

Reply:
Uhm, in case, that asset is not being imported or used in any file, you can delete it as it is not being used anywhere.

Reply:
Oh! thanks @makunomark  and @brianwachira i confirmed it was actually not being used and removed it as recommended.

Reply:
Okay, @mathewTheGreat you can push the commit  that  actually removes the file in case you haven't yet 😅